src/content/previews/using-github-actions.mdx

```diff
@@ -55,6 +55,16 @@ jobs:
           name: pr-${{ github.event.number }}
           timeout: 15m
           scope: global
+
+      # optional
+      # Run integration tests against your newly create preview environment
```

Comment:
This is what I had in mind too. But we need to add the `test` section to the okteto manifest in the repo